extra/patterns/index.js

```diff
@@ -1,4 +1,10 @@
+<<<<<<< HEAD
+import creatediamond from './diamond';
+import createequilatral from './equilateral';
+
+=======
 import creatediamond from './diamond';
 import createequilatral from './equilateral';
 
+>>>>>>> 146c3daaffaa716bb8480bc3e09ba73da5e7895f
```

Comment:
Remove unnecessary/duplicate code

Reply:
I removed all the unnecessary file and duplicate code which is present in that files.